src/pages/Login.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import TextInput from '../components/TextInput';
+import Button from '../components/Button';
+import { supabase } from '../App';
+
+const Login = () => {
+  const [email, setEmail] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+
+  const submitToSupabase = async () => {
+    const { data, error } = await supabase.auth.signInWithPassword({
```

Comment:
I'm thinking we refactor this to `then` `catch` syntax for the promise. It reads better and separates the flows. rather than give us data and error, which if it is successful, error would be empty.